server/services/voip/connector/asterisk/ami/AMIConnection.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Class representing AMI connection.
+ * @remarks
+ * This class is based on https://github.com/pipobscure/NodeJS-AsteriskManager
+ * which is internally based on https://github.com/mscdex/node-asterisk
+ *
+ * Asterisk AMI interface is a socket based interface. The AMI configuration
+ * happens in /etc/asterisk/manager.conf file.
+ *
+ */
+import { IConnection } from '../IConnection';
+import { Logger, LogLevel } from '../../../../../../lib/Logger';
+
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Manager = require('asterisk-manager');
```

Comment:
Can't we `import` it? 👀

Reply:
I tried doing that (like this `import * as Manager from 'asterisk-manager';` ) but then found out that this package doesn't provide any typescript definitions :disappointed: So I think if we want to convert it to import, we'll have to write the typescript definitions on our own which might be a little out of scope for this project :sweat_smile:

Reply:
Unfortunately there are no typedefs for this. We will have to keep it as this. May be we can create a type in another PR.
I will add a detailed comment about why. WDYT?